app/models/matchup.rb

```diff
@@ -43,7 +45,8 @@ class Matchup < ApplicationRecord
 
   has_many :picks, dependent: :restrict_with_exception
 
-  enum conference: {east: 0, west: 1, finals: 2}
+  enum sport: {nba: 0, mlb: 1}
+  enum conference: {east: 0, west: 1, finals: 2, AL: 3, NL: 4, WS: 5}
```

Comment:
See if you can add a validation to make sure `conference` matches `sport`. Look at the two examples above that use `validate do`

Reply:
Or you can do it with an `inclusion` validation like some of the others.

Reply:
added this in commit 47ee113